app/utils/kcidb.py

```diff
@@ -136,6 +136,12 @@ def push_build(build_id, first, bq_options, db_options={}, db=None):
             build[models.FILE_SERVER_RESOURCE_KEY],
             utils.BUILD_LOG_FILE
         ]),
+        'config_name': build[models.DEFCONFIG_FULL_KEY],
+        'config_url': '/'.join([
```

Comment:
It doesn't seem to be producing the expected URL, from the staging log:
```
"config_url": "https://storage.kernelci.org//kernel.config/build.log",
```

Reply:
Oops, fixed to use file_server_resource + "kernel.config".

Note however that the base URL looks wrong for staging, even for the log_url  It's using "storage.kernelci.org" and 'kernelci/staging.kernelci.org` doesn't exist there, but does on staging-storage.kernelci.org.

Reply:
OK got this now:
```
Jan 29 09:17:59 kernelci-staging kernelci-celery[20524]:       "config_url": "https://storage.kernelci.org//kernelci/staging.kernelci.org/staging-20200129.0/riscv/defconfig/gcc-8/kernel.config",
```
And yes you're right, it's not the correct storage URL for staging.  Also it has a double `//` which is probably benign but better to avoid it if we can.
I'll check why it's not using the correct storage staging URL.  Then depending on the other places where STORAGE_URL is used, we may or may not remove the trailing slash from it.
Another way to avoid a double slash (and probably safer anyway) is to use something like `urlparse.urljoin`: https://docs.python.org/2/library/urlparse.html#urlparse.urljoin

Reply:
@khilman Please use `urlparse.urljoin()` rather than the plain string join method, this will take care of removing double slashes.